src/lib/common/tag.h

```diff
@@ -156,4 +159,6 @@ extern std::string startArray
 */
 extern std::string endArray(const std::string& indent, const std::string& tagName, Format format);
 
+
```

Comment:
glups, that file should not be even referenced, slipped from a former approach using JSON_VALUE. I'll restore it